docs/source/researchers-manage-studies.rst

```diff
@@ -179,4 +179,8 @@ For external studies, the "Edit Study Design" page allows you to edit the follow
 
 .. _`Setup for custom frame development`: frame-dev-setup.html
 
-.. _`Setting study fields`: researchers-set-study-fields.html
\ No newline at end of file
+.. _`Setting study fields`: researchers-set-study-fields.html
+
+.. _`study list page`: https://childrenhelpingscience.com/exp/studies/
```

Comment:
Do know if these links names/references are case-sensitive? In "study list page", the "s" is lower case here and upper case where it's referenced in the text above. I had a quick look in the sphinx rst docs here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks but couldn't find a quick answer.